docs/api/http.md

```diff
@@ -901,7 +901,7 @@ Set a custom idle timeout for individual requests:
 
 ```ts
 const server = Bun.serve({
-  fetch(req, server) {
+  fetch: async (req, server) => {
```

Comment:
im pretty sure `async fetch(req, server) {` also works too - not sure which looks the best though